README.md

```diff
@@ -28,11 +28,34 @@ routes.add('about', '/about-us/:foo(bar|baz)', 'index')
 ```
 This file is used both on the server and the client.
 
-API: `routes.add(name, pattern, page = name)`
+API: `routes.add(name, pattern, page = name, middleware)`
 
 - `name` - The route name
 - `pattern` - Express-style route pattern (uses [path-to-regexp](https://github.com/pillarjs/path-to-regexp))
 - `page` - Page inside `./pages` to be rendered (defaults to `name`)
+- *optional:* `middleware` - One or many functions to process server-side, one by one, as long as they return a query object or promise.
```

Comment:
Agreed. Let's do some more readme optimizations in another PR to keep it consistent and simple